### DISCUSSION TITLE
Calendar Year/Month/date select

### DISCUSSION THREAD

**Discussion by technosprout** [taiga-family/taiga-ui#1543] [Ideas]
My suggestion for Calendar for quickly selecting a month/year/date

The Year when Clicked brings up a list of years (2 Clicks)

Selecting the month is not easy. Consider changing the behavior  to brining up the months when the month is clicked rather than navigating using the < > to get to the month. On average that needs 6 clicks to get to the correct month

So in about 4 clicks the user is able to get year/month/date

**Comment by sunanan** [comment]
I also want to know why this feature has not been added. I know this may have something to do with the developer's design language, but if you add it, it's really perfect

**Reply by waterplea** [reply]
This was passed down to the design team, we will see what they come up with in updated specs for datepickers.
